src/Makefile.am

```diff
@@ -1,3 +1,12 @@
 AM_CFLAGS = -Wall -Werror
-bin_PROGRAMS=init
-init_SOURCES=init.c jsmn.c net.c util.c parse.c parson.c container.c exec.c event.c portmapping.c
```

Comment:
Ooops, pushed the wrong branch, the new update should have the correct commmits.